tests/integration_tests/build/test_coverage.py

```diff
@@ -23,9 +23,9 @@
 # Checkout the cpuid crate. In the future other
 # differences may appear.
 if utils.is_io_uring_supported():
-    COVERAGE_DICT = {"Intel": 82.92, "AMD": 82.25, "ARM": 82.57}
+    COVERAGE_DICT = {"Intel": 82.92, "AMD": 82.25, "ARM": 82.47}
```

Comment:
Hmmm, what's the reason for the ARM coverage dropping? I don't see anything obvious? Is there any new unit-testing we can add?

Reply:
Before rebasing onto main, the coverage needed to be adjusted upward, so I think this is another instance of spurious coverage changes on ARM, similar to those we had when we updated linux-loader to 0.8.1 in #3321.